langlib/lang.annotations/src/main/ballerina/src/lang.annotations/annotations.bal

```diff
@@ -15,7 +15,7 @@
 // under the License.
 
 # Denote that the return value is tainted.
-public const annotation tainted on parameter, return, source listener, source var;
+public const annotation tainted on parameter, return, source listener, source var, source type;
```

Comment:
This will allow using `<@tainted>` too right? I hope that's intentional? Do we have tests for that flow?

Something like
```ballerina
public function main() {
    int x = 1;
    foo(<@tainted> x);
}

function foo(@untainted int i) {
    
}
```

Reply:
We already had logic [1] in taint analyzer to untaint for `<@untaint>`, looks like we don't have any test for it, I'll add.

[1] - https://github.com/ballerina-platform/ballerina-lang/issues/23793

Reply:
Test case added